qir/cli-tests/BUILD

```diff
@@ -0,0 +1,15 @@
+load("@rules_python//python:defs.bzl", "py_test")
+
+py_test(
+    name = "test-profile-cli-config",
+    srcs = [
+        "test_profile_cli_config.py",
+    ],
+    args = [
+        "$(location //qir/qat:qat)",
+    ],
+    data = [
+        "//qir/qat",
+    ],
+    main = "test_profile_cli_config.py",
+)
```

Comment:
It is undesirable that 
* there is a dash-separated (`test-profile-cli-config`) and underscore-separated (`test_profile_cli_config`) identifier for the same thing. This encourages a confusion, and sometimes the bugs hard to catch. Consider using the same identifier for all cases;
* the same string literal ("test_profile_cli_config.py") occurs more than once, consider de-duplicating.

Reply:
OK.